tests/test_bsuite.py

```diff
@@ -112,15 +112,56 @@ def test_seeding(env_id):
     env_2.close()
 
 
-@pytest.mark.parametrize("env_id", BSUITE_ENV_IDS)
+# Without EzPickle:_register_bsuite_envs.<locals>._make_bsuite_env cannot be pickled
+# With EzPickle: maximum recursion limit reached
+FAILING_PICKLE_ENVS = [
+    "bsuite/bandit_noise-v0",
```

Comment:
I think I have found this before. It is generally because one of the classes implemented `__getattr__`. 
Is there a particular parameter that is missing? 

I suspect the issue is that the environment only defines a variable on `reset` or another function

Reply:
I’ll go check that individual env and see. But what would we be able to do to fix it, besides submitting a PR to their repo? I guess we could in the compatibility wrapper specifically check if it’s that env or if an env has that specific variable not defined in init, and then do whatever modifications are required?

Reply:
In short, we can't fix it. The environments failing use a wrapper that contains
```python
def __getattr__(self, attr):
    return getattr(self._env, attr)
```

The problem exists when `_env` doesn't exist in the wrapper, i.e., when a staticmethod is existed (`__setstate__`) then this causes an infinite loop to occur.
The second issue is that dm don't seem to be maintaining the project anymore.

Reply:
Could you put a PR up by any chance? Even if they don't' end up merging it I feel like we might as well try, you seem to understand this stuff better than me though, I'm not sure I'd be able to explain it well or respond to any questions about it.